src/hotspot/share/gc/shenandoah/shenandoahHeap.hpp

```diff
@@ -309,7 +309,7 @@ class ShenandoahHeap : public CollectedHeap {
     WEAK_ROOTS_BITPOS  = 4,
 
     // Old regions are under marking, still need SATB barriers.
-    OLD_MARKING_BITPOS = 5
+    OLD_MARKING_BITPOS = 5,
```

Comment:
Are these trailing commas intentional?

Reply:
my mistake.  I had another constant in this list, and then removed it.

Reply:
fixed.